core-services/prow/02_config/_plugins.yaml

```diff
@@ -1068,12 +1062,6 @@ config_updater:
         - ci
       gzip: true
       name: job-config-misc
-    ci-operator/jobs/identitatem/**/*-release-0.1*.yaml:
-      clusters:
-        app.ci:
-        - ci
-      gzip: true
-      name: job-config-misc
```

Comment:
I removed it because I was getting an error in correctly-sharded-config and this was the only way I could get the job to not fail.  See all the failures above.  Tried to get help via slack on what the problem was but no replies.